src/pages/gateway/extending-unified-schema.md

```diff
@@ -60,7 +61,7 @@ In this example, we will use `additionalResolvers` to apply a set of discounts t
     "files": [
```

Comment:
The files array may not be needed on the mesh config, since the CLI handles it. @revanth0212 can you confirm?

Reply:
Yes @prabhuram93. 

We can update the documentation to represent the same. We can just have `./additional-resolvers.js` in the mesh config and have a file called `additional-resolvers.js` with the contents in the same folder.

Reply:
@revanth0212 if this should be handled in a separate file, please provide the updated mesh config.